presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -104,5 +104,25 @@ The following properties allow the configuration of remote function execution:
 
     The UDS (unix domain socket) path to communicate with a local remote
     function server. If specified, takes precedence over
-    ``remote-function-server.thrift.address`` and 
+    ``remote-function-server.thrift.address`` and
     ``remote-function-server.thrift.port``.
+
+JWT authentication support
+--------------------------
+
+Prestissimo supports JWT authentication for internal communication.
+For details on the generally supported parameters visit :doc:`JWT</security/internal-communication>`.
```

Comment:
```suggestion
For details on the generally supported parameters visit `JWT </security/internal-communication.html#jwt>`_.
```

Links directly to the JWT section.

Reply:
Not sure the proposed change generates the desired link (it is changed to an external link). Locally, it creates `file:///security/internal-communication.html#jwt` which is not correct.

Reply:
Interesting: I built locally with this proposed change (yesterday, and just now to retest after your comment) and the link works for me, opening the page at the JWT target. 

If it doesn't work for you, please ignore this suggestion and I'll look into this later. Thanks for checking!

Reply:
This works
```
`JWT <../security/internal-communication.html#jwt>`_.
```
(relative path needed the .. to switch dirs properly).
I will make that change.